pvlib/iotools/pvgis.py

```diff
@@ -424,6 +441,13 @@ def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
         base url of PVGIS API, append ``tmy`` to get TMY endpoint
     timeout : int, default 30
         time in seconds to wait for server response before timeout
+    utc_offset: int, default 0
+        Use to specify a time zone other than the default UTC zero and roll
+        dataframe by ``utc_offset`` so that it starts at midnight on January
+        1st. If not zero, will also force year to ``coerce_year``.
+    coerce_year: int, default 1990
```

Comment:
Let's do `coerce_year=None` instead, for consistency with the other pvlib functions that have `coerce_year` (`read_epw`, `read_tmy3`).  

If we do have a default year, that would be a change in behavior that could warrant a deprecation period...

Reply:
I think if we make both defaults `None`I can check identity, add a comment to avoid implicit booleaness, and then make it possible to coerce the year even for utc zero. I will send an update later today.

I prefer `utc_offset` over `roll_offset` because this function converts the time zone to a new utc offset, which is not explained in “roll offset” while utc offset is a well known concept. How about `roll_utc_offset` which unfortunately sounds like a verb, but makes some sense if I let the default be `None`?